collect_app/src/main/java/org/odk/collect/android/widgets/ExStringWidget.java

```diff
@@ -265,6 +265,7 @@ public void onButtonClick(int buttonId) {
             }
         } else {
             onException(errorString);
+            setFocus(getContext());
```

Comment:
This change looks strange to me. I would rather expect something like https://stackoverflow.com/a/5106399 Could you explain that `setFocus`?
and even if it' ok it should be in `onException()` method I think.